tsconfig.eslint.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Why do we need it? Does `"eslint": "eslint --ext mjs,ts,tsx --max-warnings 0 -f codeframe --cache --color src stories jest tools",` not work?

Reply:
This one's for the `parserOptions.project` setting. Without this the type-aware typescript-eslint rules can't work.